### PR TITLE
split send-email

### PR DESCRIPTION
**Pull request by david-deboer**
This was going to be a bigger PR to add some other watch dogs that I think are somewhat deprecated in the new scheme.  However, I did want to keep the "mail" method broken out from the initial, so this is to do that.

### CODE REVIEW

File: hera_mc/watch_dog.py [hunk 1]
```diff
@@ -6,6 +6,40 @@
 """System watch-dogs."""
 
 
+def send_message(subject, msg, to_addr=None, from_addr='hera@lists.berkeley.edu', skip_send=False):
```
Comment (dannyjacobs) [inline hera_mc/watch_dog.py:9]: send_email might be more accurate. Is it ever not an email? Like a SMS or slack?

**Review by dannyjacobs — COMMENTED** [review]
Making things difficult for myself.

File: hera_mc/watch_dog.py (hunk 1 above)
Comment (david-deboer) [reply hera_mc/watch_dog.py]: I honestly have no preference, but you are right that that method is for email, so I've renamed.

**Comment by david-deboer** [comment]
The actual purpose of this PR has been superseded, but I want to retain breaking out the send-email functionality.

**Review by bhazelton — APPROVED** [review]
Looks fine to me, but I am not familiar with the smtp stuff.

**Pull request merged**